manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml

```diff
@@ -22,6 +22,14 @@ rules:
     verbs: ["*"]
```

Comment:
This still exists, it still has high access permission and can get all SCC.
```console
- apiGroups:
  - '*'
  resources:
  - '*'
  verbs:
  - '*'
 ```

Reply:
@jianzhangbjz we need [this PR](https://github.com/openshift/operator-framework-olm/pull/600) which contains [this commit](https://github.com/openshift/operator-framework-olm/pull/600/commits/43f13e1c239bc980642259ca29b624c15a35db23).

Reply:
I've pulled that PR into this commit.

Reply:
Got it, thanks!